lib/NetworkManager.js

```diff
@@ -302,6 +312,12 @@ class Request {
     this._headers = {};
     for (const key of Object.keys(payload.headers))
       this._headers[key.toLowerCase()] = payload.headers[key];
+
+    this._fromMemCache = false;
+  }
+
+  _setFromMemoryCache(fromMemCache) {
```

Comment:
nit: let's drop the method and reach into request right away

Reply:
done